apis/fluentbit/v1alpha2/clusteroutput_types.go

```diff
@@ -43,6 +43,8 @@ type OutputSpec struct {
 	// A user friendly alias name for this output plugin.
 	// Used in metrics for distinction of each configured output.
 	Alias string `json:"alias,omitempty"`
+	// AzureBlob defines AzureBlob Output Configuration
+	AzureBlob *output.AzureBlob `json:"azureblob,omitempty"`
```

Comment:
Would it be better to use the hump rule here? `azureBlob`?

Reply:
I'm not sure, I based it on what is currently in the output types (see `OpenSearch`, `DataDog`). None of the other outputs use camelCase, but happy to change it 😄